nym-connect/desktop/src-tauri/src/models/mod.rs

```diff
@@ -112,3 +113,9 @@ pub struct PagedResult<T> {
     pub total: i32,
     pub items: Vec<T>,
 }
+
+#[cfg_attr(test, derive(ts_rs::TS))]
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct Gateway {
+    pub identity: String,
+}
```

Comment:
If the purpose is to return identity address of a gateway, why declaring a whole type for it instead of just using primitive `String` directly ? Do we expect adding more fields to the type in the future ?

Reply:
Yep I expect we will add more fields here